static/app/views/alerts/rules/issue/details/alertChart.tsx

```diff
@@ -176,13 +176,13 @@ const FooterHeader = styled(SectionHeading)`
   line-height: 1;
 `;
 
-const FooterValue = styled('div')`
+export const FooterValue = styled('div')`
   display: flex;
   align-items: center;
   margin: 0 ${space(1)};
 `;
 
 /* Override padding to make chart appear centered */
-const StyledPanelBody = styled(PanelBody)`
+export const StyledPanelBody = styled(PanelBody)`
```

Comment:
i'd rather these be copied than exported since the two layouts don't have much in common